codegen/sdk-codegen/build.gradle.kts

```diff
@@ -208,13 +193,6 @@ task("stageSdks") {
                 from("${it.outputDir}")
                 into("${it.sourcesDir}")
                 exclude("Package.swift")
```

Comment:
If you comment this line, the service clients will be written to the SDK as individual Swift packages.